### ISSUE TITLE
Accomodate !!Con-style submissions with multiple required fields

### ISSUE THREAD

**Issue by blinry** [pretalx/pretalx#950]
## Problem you are facing

When accepting talk submissions, we're following a !!Con-style approach where we ask for an abstract, a timeline (detailing how the 10 min will be spent), and an intended audience. We were able to add the latter two via the "questions" feature, but they are displayed on a second page.

This is inconvenient for the submitter: they don't have a good overview of the required fields, and they can't check them out without typing something in the (required) "abstract" field. When working on the three fields in parallel, they need to navigate back and forth.

## Possible Solution

My suggestion would be to put the "regular" fields as well as the "extra" questions on the same page. I don't see any downsides to this, but I'd be happy to discuss them, if someone thinks there are any?

The extra questions should also be displayed in the reviewers' "please review this submission" dialog, without them having to click on the tabs at the top.

## Context

I *think* this is the last "please accommodate our weird EnthusiastiCon needs" feature request for now. :D

**Comment by rixx** [comment]
Questions are currently on a separate page because you can restrict them by track or submission type – so we need that information first, and duplicating the logic in the frontend to show/hide the questions selectively sounds like a very unreliable way of handling things to me. I'd be happy to discuss ideas for this – JS code generation, maybe? But if no ideas come up, I'll probably close this issue.

> The extra questions should also be displayed in the reviewers' "please review this submission" dialog, without them having to click on the tabs at the top.

Absolutely, I added this change at least for the submission questions. Should make the review process much less painful.

> I think this is the last "please accommodate our weird EnthusiastiCon needs" feature request for now. :D

Your feedback is super appreciated! I hope we can find ways to accommodate your format/wishes/requests. I'm sure I don't always see potential applications of features, or solutions to problems, and I'm happy for any feedback.

**Comment by rixx** [comment]
I'm closing this issue as out-of-scope for now: We can revisit it if we ever get around to making the submission process embeddable (aka re-implementing the whole monster in pure JS). If we do that, we can also offer it as a single page process or as something with arbitrary ordering.
